core/sql/sqlcomp/CmpSeabaseDDLschema.cpp

```diff
@@ -1006,7 +1006,7 @@ void CmpSeabaseDDL::alterSeabaseSchema(StmtDDLAlterSchema * alterSchemaNode)
 
    bool isVolatile = (memcmp(schName.data(),"VOLATILE_SCHEMA",strlen("VOLATILE_SCHEMA")) == 0);
    int32_t length = 0;
-   int32_t rowCount = 0;
+   Int64 rowCount = 0;
```

Comment:
This rowCount is used at line 889. It is a lengthy method , hence not visible unless file is expanded. Thanks.